src/components/RunMap/index.jsx

```diff
@@ -38,12 +36,10 @@ const RunMap = ({
         const map = ref.getMap();
         if (map && IS_CHINESE) {
           map.addControl(new MapboxLanguage({ defaultLanguage: 'zh-Hans' }));
-          if (RoadLabelDisplay == false) { 
-            // todo delete layers
-            map.on('load', () => {
-              for(let layerId of layerList){
-                map.removeLayer(layerId);
-              }
+          if (!ROAD_LABEL_DISPLAY) { 
+          // todo delete layers
+          map.on('load', () => {
+              MAP_LAYER_LIST.forEach((layerId) => {map.removeLayer(layerId)})
```

Comment:
add two spaces for this lines

Reply:
make it looks better and it works fine, but i don't it's right or not